src/daffodilDebugger.ts

```diff
@@ -104,42 +101,25 @@ export async function getDebugger(config: vscode.DebugConfiguration) {
 
       // Code for downloading and setting up daffodil-debugger files
       if (!fs.existsSync(`${rootPath}/${artifact.name}`)) {
-        // Get daffodil-debugger of version entered using http client
-        const client = new HttpClient('client') // TODO: supply daffodil version from config
-        const artifactUrl = artifact.archiveUrl(backend)
-        const response = await client.get(artifactUrl)
-
-        if (response.message.statusCode !== 200) {
-          const err: Error = new Error(
-            `Couldn't download the Daffodil debugger backend from ${artifactUrl}.`
+        // Get daffodil-debugger zip from extension files
+        const filePath = path
+          .join(
+            context.asAbsolutePath('./server/core/target/universal'),
+            artifact.archive
           )
-          err['httpStatusCode'] = response.message.statusCode
-          throw err
+          .toString()
+
+        // If debugging the extension without vsix installed make sure debugger is created
+        if (!filePath.includes('.vscode/extension')) {
+          if (!fs.existsSync(filePath)) {
+            let baseFolder = context.asAbsolutePath('./')
+            child_process.execSync(
+              `cd ${baseFolder} && sbt universal:packageBin`
```

Comment:
This feels wrong to me. This seems like it should all be part of the build process and not part of the extension it self.